packages/webui/src/components/ui/TraceDetail.tsx

```diff
@@ -20,8 +20,14 @@ import ResponseHeaders from './ResponseHeaders';
 import { TabContent, TabList, TabListItem } from './Tabs';
 import TimingsDiagram from './TimingsDiagram';
```

Comment:
Also not quite related to these code changes, but for the "Aborted" and "Timeout" examples, maybe we shouldn't include empty the "Response Headers" and "Timing" sections:

![image](https://github.com/FormidableLabs/envy/assets/17857833/c2d6017d-fe3e-4040-8196-278cdd5b0ae2)